### PR TITLE
change: app title

### PR DESCRIPTION
**Pull request by Till1983**
I changed the app title in app-config.yaml to "CODE Internal Developer Platform".

### CODE REVIEW

**Comment by Till1983** [comment]
@MennaTullahTaha I completely forgot to tag you here. I hope it's not too late for a review.

File: app-config.yaml
Comment (MennaTullahTaha) [inline app-config.yaml:1]: @juliusdw @DrAdamRoe 
As far as I understand the e2e test expects Scaffold Backstage App to pass the tests, wouldn't that break the e2e? someone will need to make a change I assume.

**Review by MennaTullahTaha — CHANGES_REQUESTED** [review]

File: app-config.yaml
Comment (juliusdw) [reply app-config.yaml:1]: yes, the e2e test need to be changed as follows:
change this line in `app/e2e-tests/app.test.ts`
from
`await expect(page.getByText('Scaffolded Backstage App')).toBeVisible();`
to
`await expect(page.getByText('CODE Internal Developer Platform')).toBeVisible();`

Comment (Till1983) [reply app-config.yaml:1]: @MennaTullahTaha @DrAdamRoe  
I changed the e2e test in `packages/app/e2e-tests/app.test.ts`, just like @juliusdw pointed out. The title is now consistent with the change to the title that I made in `app-config.yaml`. I'm sorry for the long delay!

**Pull request closed without merging**